onnx2kerastl/customonnxlayer/onnxless.py

```diff
@@ -0,0 +1,23 @@
+import tensorflow as tf
+from keras.layers import Layer
+
+
+class OnnxLess(Layer):
```

Comment:
Why do we need a custom Onnx layer for this? if you create this in the following way, you might need to propagate these changes to the UI (might be time consuming).